modules/nxiq-pg-db/main.tf

```diff
@@ -34,20 +34,64 @@ terraform {
 }
 
 # --------------------------------------------------------------------------
-# Create a unique database for NXIQ
+#
+# Copyright 2023-Present Sonatype Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# --------------------------------------------------------------------------
+
+# --------------------------------------------------------------------------
+# Require a minimum version of Terraform and Providers
 # --------------------------------------------------------------------------
+terraform {
+  required_version = ">= 1.4.5"
```

Comment:
@madpah the required providers are duplicated causing issues

Reply:
Duplicated against where?